spec/data-model/README.md

```diff
@@ -214,11 +220,44 @@ that the implementation attaches to that _annotation_.
 ```ts
 interface UnsupportedAnnotation {
   type: "unsupported-annotation";
-  sigil: "!" | "@" | "#" | "%" | "^" | "&" | "*" | "<" | ">" | "/" | "?" | "~";
+  sigil: "!" | "@" | "%" | "^" | "&" | "*" | "+" | "<" | ">" | "?" | "~";
   source: string;
 }
 ```
 
+## Markup
+
+Each of the markup interfaces `MarkupOpen`, `MarkupStandalone`,
+and `MarkupClose` represent the _markup_ `type` of the corresponding
+`kind`.
```

Comment:
I don't know how to read this sentence... what is it trying to communicate? Perhaps something like this?
```suggestion
A `Markup` object is either `MarkupOpen`, `MarkupStandalone`, or `MarkupClose`,
which are differentiated by `kind`.
```